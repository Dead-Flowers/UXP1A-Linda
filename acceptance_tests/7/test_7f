-t 1 output '(1)'
-t 2 output '(1)'
-t 3 output '(1)'
-t 4 output '(1)'
