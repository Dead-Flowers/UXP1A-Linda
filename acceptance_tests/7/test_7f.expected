Exit code: 0
Exit code: 0
Exit code: 0
Exit code: 0
